pom.xml

```diff
@@ -22,13 +22,13 @@
     <url>https://github.com/quarkiverse/quarkus-cucumber</url>
   </scm>
   <properties>
-    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <compiler-plugin.version>3.11.0</compiler-plugin.version>
```

Comment:
This is inherited from quarkiverse-parent. It can be safely removed (including the plugin declaration)

Reply:
@gastaldi This is done on the last update

Reply:
Actually the declaration in the `runtime` module is necessary, just not in the pluginManagement, sorry for not being clear

Reply:
I just removed it and the build is failing